openwpm/deploy_browsers/configure_firefox.py

```diff
@@ -1,7 +1,103 @@
 """ Set prefs and load extensions in Firefox """
 
+import json
+import re
+from pathlib import Path
+from typing import Any, Dict
+
+# TODO: Remove hardcoded geckodriver default preferences once
```

Comment:
We should add a step into our [release checklist](https://github.com/mozilla/OpenWPM/blob/master/docs/Release-Checklist.md) to check whether these prefs should be updated whenever a new version of geckodriver is released. We should also have an issue on file to remove the hack once the fix ships.

Reply:
Done! Here's the issue: https://github.com/mozilla/OpenWPM/issues/867